local/docker-compose.example.yml

```diff
@@ -0,0 +1,26 @@
+version: '3'
+
+services:
+  database:
```

Comment:
please rename to, for eg postgres or postgresql

Reply:
done